ops/roles/ota/tasks/main.yml

```diff
@@ -36,6 +37,7 @@
     force: yes
     accept_hostkey: yes
     key_file: '/home/{{ ansible_user }}/.ssh/ota-bot-key'
+    depth: 1
```

Comment:
For reference for other reviewers

```Create a shallow clone with a history truncated to the specified number or revisions. The minimum possible value is 1, otherwise ignored. Needs git>=1.9.1 to work correctly.```

Reply:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/git_module.html